Bottom: cda58795298ab3fd6eaa9cc5c32fa407527d2f56
Top:    3c80ce2a67baeab96fd06c350716dccc4bb7ad72
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 14:42:13 +0800

Refresh of autofs-5.0.7-document-browse-option-in-man-page.patch

---

diff --git a/man/auto.master.5.in b/man/auto.master.5.in
index 72766a2..c552e56 100644
--- a/man/auto.master.5.in
+++ b/man/auto.master.5.in
@@ -153,7 +153,10 @@ so is given without a leading dash. Use of the browse option pre-creates
 mount point directories for indirect mount maps so the map keys can be
 seen in a directory listing without being mounted. Use of this option
 can cause performance problem if the indirect map is large so it should
-be used with caution.
+be used with caution. The internal program default is to enable browse
+mode for indirect mounts but the default installed configuration overrides
+this by setting BROWSE_MODE to "no" because of the potential performance
+problem.
 .TP
 .I "nobind"
 This is an autofs specific option that is a pseudo mount option and
